src/Uno.UI/Microsoft/UI/Xaml/Controls/NavigationView/NavigationViewItemBase.cs

```diff
@@ -1,13 +1,34 @@
-﻿// MUX reference NavigationViewItemBase.cpp, commit f3da94f 
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+// MUX reference NavigationViewItemBase.cpp, commit 574e5ed 
 
+using Uno.UI.Helpers.WinUI;
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
-using Windows.UI.Xaml.Media;
 
 namespace Microsoft.UI.Xaml.Controls
 {
 	public partial class NavigationViewItemBase : ContentControl
 	{
+		protected override void OnApplyTemplate()
+		{
+			base.OnApplyTemplate();
+
+			// TODO Uno specific - unsubscribe Loaded event handler to avoid multiple subscriptions
+			// as OnApplyTemplate will be called repeatedly.
+			Loaded -= OnLoaded;
+			Loaded += OnLoaded;
```

Comment:
Why isn't this placed in the ctor instead ? Is it the original location in C++?

Reply:
It is placed in `OnApplyTemplate` in MUX as well (apparently it resolves a specific edge case). We don't have this issue now, but we could have when we have the control lifecycle aligned